Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ruby:2.1
+
+MAINTAINER PyConUK Organisers
+# Based heavily on grahamc/jekyll
+
+RUN apt-get update \
+    && apt-get install -y node python-pygments wget unzip \
+    && apt-get clean \
+    && gem install \
+        bundler \
+        html-proofer \
+        jekyll \
+        jekyll-redirect-from \
+        kramdown \
+        rdiscount \
+        rouge \
+        rvm \
+    && wget https://github.com/PyconUK/pyconuk.github.io/archive/master.zip \
```

Comment:
We shouldn't need to pull down the code base here, it should be running in the cloned repo already. I think this route would also cause issues with making local changes.